pandas/core/indexes/range.py

```diff
@@ -380,8 +380,14 @@ def get_loc(self, key, method=None, tolerance=None):
 
     @Appender(_index_shared_docs["get_indexer"])
     def get_indexer(self, target, method=None, limit=None, tolerance=None):
-        if not (method is None and tolerance is None and is_list_like(target)):
-            return super().get_indexer(target, method=method, tolerance=tolerance)
+        if not (
+            method is None
```

Comment:
Maybe a little more concise to use `com.all_none` here instead of doing individual checks:
```python
if not (com.all_none(method, tolerance, limit) and is_list_like(target)):
```
Or could expand and negate:
```python
if com.any_not_none(method, tolerance, limit) or not is_list_like(target):
```
No strong feelings here though, so feel free to choose either of the above or keep the existing pattern.

Reply:
thanks!